src/atoms/Button/Button.js

```diff
@@ -182,7 +219,7 @@ const StyledTag = createStyledTag(name, props => ({
 class Button extends Component<ButtonProps> {
   static defaultProps = {
     ...theme[name].defaults,
-    size: 'md',
+    ...DEFAULT_PROPS,
```

Comment:
Needs refactor otherwise we will get confused with `DEFAULT_PROPS` and `theme[name].defaults`

Reply:
@ej9x fixed